membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala

```diff
@@ -177,8 +177,8 @@ object ProductRatePlanChargeProductType {
     override val id = "Tier Three"
   }
 
-  object GuardianLight extends ProductRatePlanChargeProductType {
-    override val id = "Guardian Light"
+  object GuardianAdLite extends ProductRatePlanChargeProductType {
```

Comment:
To follow precedent I think `AdLite` would be enough, similar to `Weekly`.  AFAIK the only reason we had to add `GuardianPatron` is because we already have `Patron` for the membership version

Reply:
The Portfolio team have decided to not make these change as it would require lots of downstream systems to be updated in the process.

Reply:
fair enough if that's decided, but it's just a scala case class in one project isn't it?  Unless that would require renaming the API name and in zuora etc as well which I agree might be a bit much by this time.

Reply:
It's a valid comment. But based on the fact that we will not change it everywhere, I don't see the point in updating just the Scala case class when the string still refers the old name?

Reply:
to be honest it was really just to declutter and make it consistent with Weekly above but I can also see the value of using the full name everywhere so if you're happy then I'm happy!
https://github.com/guardian/members-data-api/blob/38b056bd7df882630bbc4140386e7701b686ff9f/membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala#L172-L174

Reply:
I think it's a good compromise to update just the case class.

It is updated here: https://github.com/guardian/members-data-api/pull/1111/commits/21093cd1d6ac456c97d9bc8700902a3d6c5bf3d5